i3pystatus/openvpn.py

```diff
@@ -54,7 +54,13 @@ class OpenVPN(IntervalModule):
     def init(self):
         if not self.vpn_name:
             raise Exception("vpn_name is required")
-
+        
+        # use new service names if openvpn version is 2.4 (2.5 is not out yet...)
+        if run_through_shell('openvpn --version | grep -i "openvpn 2.4"', True).out.strip():
```

Comment:
that's true, I didn't think about that.
Do you think it's worth checking if the command is not set, then override it?
like `if self.vpn_down_command == <insert old value>`
but this is super messy